earthpy/spatial.py

```diff
@@ -254,7 +254,7 @@ def _stack_bands(sources, write_raster=False, dest=None):
         for src in sources:
             src.profile
 
-    except ValueError as ve:
+    except Exception as e:
```

Comment:
@joemcglinchy i think we want to raise a value error here given that is what we are trying to catch. i might be missing something??

Reply:
with my test of passing in two numpy arrays in a list as the argument to `_stack_bands`, the numpy ndarray object doesn't have a profile attribute, so an `AttributeError` is raised. I think this would be similar to any other input that is contained in a list.

Reply:
oh i see that. Yes so let's change them BOTH to attribute error then to be specific! what do you think?